df.itemimprovements.xml

```diff
@@ -191,7 +191,7 @@
         <stl-vector name='styles' type-name='written_content_style'/>
         <stl-vector name='style_strength' type-name='int32_t' comment='0 = maximum, 1 = significant, 2 = partial'/>
         <int32_t name='author' ref-target='historical_figure'/>
-        <int32_t init-value='-1'/>
+        <int32_t name='author_roll' init-value='-1'/>
```

Comment:
This one is named `writing_quality` on master at the moment. Any reason for this name instead?

Reply:
Because that's what it's named in the Legends XML export